SkatkovAM/lab_2/lab2.cpp

```diff
@@ -0,0 +1,62 @@
+
+#include <iostream>
+
+#include <ctime>
+using namespace std;
+int main() {
+    const int basicSize=10;
+    const int newSize= basicSize / 2;
+int arr[basicSize];
+    srand(static_cast<int>(time(0)));
+    cout<<"Basic array: ";
+for(int i=0; i < basicSize; i++){
+    arr[i]=rand()%10+1;
+    cout<<arr[i]<<' ';
+}
+    int i=0;
+int im_helpful;
+while (i < basicSize - 1){
+    int j=0;
+    while(j < basicSize - 1) {
+        if (arr[j] >arr[j + 1]) {
+            im_helpful=arr[j+1];
+            arr[j+1]=arr[j];
+            arr[j]=im_helpful;
+        }
+        j++;
+    }
+    i++;
+}
+
+
+
+
+cout<<endl<<"Sorted: ";
+    for(int i=0; i < basicSize; i++){
+        cout<<arr[i]<<' ';
+    }
+
+int arr2[newSize];
+
+    for(int i=0;i<newSize;i++){
+    arr2[i]=arr[2*i+1];
```

Comment:
arr[2] should consists of odd elements, not element with odd index
for given arr = {2, 3, 5, 7, 8}
arr2 should be = {3, 5, 7}